tests/test_scipy_solver.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import unittest
+
+import dimod
+
+from src.utils.scipy_solver import SciPyCQMSolver
```

Comment:
I believe @thisac and @k8culver are working on a package restructure. This was to unblock @k8culver because python-mip does not support osx very well.